sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1885,6 +1831,80 @@ def _mapping_event_from_change_event(self, status, port_dict):
         helper_logger.log_debug("mapping from {} {} to {}".format(status, port_dict, event))
         return event
 
+    # Update port sfp info and dom threshold in db during xcvrd bootup
+    def _post_port_sfp_info_dom_thr_to_db_during_xcvrd_bootup(self, port_mapping, xcvr_table_helper, stop_event=threading.Event()):
```

Comment:
rename to ? post_port_sfp_info_and_dom_thr_to_db_once()

Reply:
I have addressed this now.